### ISSUE TITLE
Feature request: live or interval-based updates

### ISSUE THREAD

**Issue by dbkaplun** [luin/medis#36]

**Comment by luin** [comment]
Could you provide more details?

**Comment by dbkaplun** [comment]
Seems like the only way to see new redis keys appear in the UI is to manually refresh with <kbd>Cmd+R</kbd>. Or am I missing something?

**Comment by luin** [comment]
That's right. There's no built-in live updates since it is not easy to implement and may have non-trivial impact on the performance.

**Comment by dbkaplun** [comment]
Yep, it would be great if there was an option to select the amount of time between refreshes, or 0 to disable!

**Comment by Gavrisimo** [comment]
Agree, this option would be cool!
